### ISSUE TITLE
Notation3 parsing : no support for blank nodes as predicate?

### ISSUE THREAD

**Issue by phochste** [rdfjs/N3.js#262]
In version 1.11.1 I get a parsing error when reading:

```
@prefix : <http://example.org/ex#>.
:Subject  [ a :Predicate ] :Object.
```

Got a

```
Error: Expected entity but got [ on line 3.
```

**Comment by RubenVerborgh** [comment]
Thanks @phochste, would you be available to test https://github.com/rdfjs/N3.js/pull/263?

**Comment by phochste** [comment]
> Thanks @phochste, would you be available to test #263?

Yes!
